ml-dhs-aws/src/main/java/com/marklogic/client/dhs/JobRunner.java

```diff
@@ -124,49 +124,33 @@ public void run(DatabaseClient client, InputStream csvRecords, InputStream jobCo
                .withBatchSize(getBatchSize());
 			if (logger != null) {
 				batcher = batcher.onBatchSuccess(batch -> {
-					if (getBatchSize() == 1) {
-						logger.info("Success for the batch-" + batch.getJobBatchNumber() + ". Total successes - 1");
-						jobRunnerVariables.lastSuccessCount.set(jobRunnerVariables.totalSuccessCount.longValue() + 1);
-						jobRunnerVariables.lastSuccessLogTime.set(System.currentTimeMillis());
-					} else {
 						long lastTime = jobRunnerVariables.lastSuccessLogTime.longValue();
 						long currentTime = System.currentTimeMillis();
 						long threshhold = lastTime + (getTimeIntervalInSeconds() * 1000);
+						jobRunnerVariables.totalSuccessCount.incrementAndGet();
```

Comment:
Good idea.  
For that matter, how about capturing the result of incrementAndGet() by setting the currentTotalSuccessCount variable outside the branch instead of inside the branch?  That way, no separate get operation is required, so no other thread has an opportunity to change the value.